eurocropsml/acquisition/region.py

```diff
@@ -162,7 +162,6 @@ def add_nuts_regions(
         geometry_df.to_file(
             geom_dir.joinpath(f"{config.country}.geojson"),
             driver="GeoJSON",
-            crs=shapefile.crs,
```

Comment:
Is this removal backward compatible, or should we specify a lower bound version on our pyogrio requirement now?

Reply:
Good point! I'll check whether it's backward compatible.

Reply:
It should be backward compatible. However, I just realized that not the `pyogrio` update caused the issue, but the `geopandas` update. Sorry for the confusion!

When saving the GeoDataFrame, older `geopandas` versions default to the `fiona` version, whereas the latest update defaults to the `pyogrio` engine. `pyogrio` does not allow for a CRS to be set, it just obtains it directly from the GeoDataFrame. Whereas `fiona` does allow for a CRS parameter and only obtains it from the GeoDataFrame if no CRS is passed. So this should be backward compatible.

Reply:
okay, thanks for checking :+1: